ios/RCTVideo.m

```diff
@@ -732,7 +732,10 @@ - (void)setProgressUpdateInterval:(float)progressUpdateInterval
 - (void)removePlayerLayer
 {
     [_playerLayer removeFromSuperlayer];
-    [_playerLayer removeObserver:self forKeyPath:readyForDisplayKeyPath];
+    @try{
```

Comment:
What's triggering this crash? I'd like to fix the issue where we remove an observer that doesn't exist rather than putting a bandaid over it with try / catch.